packages/core/index.js

```diff
@@ -1,35 +1,30 @@
 'use strict';
 
 const debug = require('debug')('untool:core');
+const isPlainObject = require('is-plain-object');
 const define = require('mixinable');
 
-const { getConfig } = require('./lib/config');
-const { environmentalize } = require('./lib/utils');
+const { getConfig, getMixins } = require('./lib/config');
 
 exports.Mixin = class Mixin {
-  constructor(config, options) {
+  constructor(config, ...args) {
```

Comment:
I would prefer not to: This would imply passing six arguments to the mixin constructor(s) - at least in server mode. I would prefer keeping things as they are for now...

Reply:
...additionally, I am partial to being able to see that, in server mode, `render` is essentially an express middleware.

Reply:
Oh, and just to be sure: in core mixins, it will be `constructor(config, options)`, while in runtime it is `constructor(config, element, options)`. You are right in assuming that options is, well, optional...